source/SpinalHDL/Design errors/hierarchy_violation.rst

```diff
@@ -5,28 +5,28 @@ Hierarchy violation
 Introduction
 ------------
 
-SpinalHDL will check that you never access a signal which is outside the current component range.
+SpinalHDL will check that signals are never accessed outside of the current component's boundaries.
```

Comment:
I think so. I didn't use it here because I figured that because range was used instead of scope originally, it implied something more or less strict than scope.